src/components/forms/PractitionerForm/UserIdSelect/index.tsx

```diff
@@ -132,6 +132,37 @@ export const UserIdSelect = (props: Props) => {
     return allUsers;
   };
 
+  /** pull all practioners */
+  const getAllPractitioners = async () => {
```

Comment:
should we move this to https://github.com/onaio/reveal-frontend/blob/master/src/containers/pages/PractitionerViews/helpers/serviceHooks.ts